autonomi/src/client/high_level/files/archive_private.rs

```diff
@@ -129,6 +129,13 @@ impl PrivateArchive {
 
         Ok(root_serialized)
     }
+
+    /// Merge with another archive
+    ///
+    /// Note that if there are duplicate entries for the same filename, the files from the other archive will be the ones that are kept.
+    pub fn merge(&mut self, other: &PrivateArchive) {
```

Comment:
this new function is not being used?
at least some E2E CI test to confirm it shall work as expected ?

Reply:
Will make a unit test right